model/project.go

```diff
@@ -315,6 +315,16 @@ func (l *ModuleList) IsIdentical(m manifest.Manifest) bool {
 	return reflect.DeepEqual(manifestModules, projectModules)
 }
 
+func (l *ModuleList) GetModuleByName(find string) (*Module, error) {
+	for _, module := range *l {
+		if module.Name == find {
+			return &module, nil
+		}
+	}
+
+	return nil, errors.New("No such module on this project.")
```

Comment:
This error message would be more helpful with the actual name: "module '<module_name>' doesn't exist"